airbyte-integrations/connectors/source-bigquery/src/test-integration/java/io/airbyte/integrations/source/bigquery/BigQuerySourceEscapeColumnNameTest.java

```diff
@@ -0,0 +1,100 @@
+package io.airbyte.integrations.source.bigquery;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.cloud.bigquery.Dataset;
+import com.google.cloud.bigquery.DatasetInfo;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.string.Strings;
+import io.airbyte.commons.util.MoreIterators;
+import io.airbyte.db.bigquery.BigQueryDatabase;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.CatalogHelpers;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.Field;
+import io.airbyte.protocol.models.JsonSchemaType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.List;
+
+import static io.airbyte.integrations.source.bigquery.BigQuerySource.CONFIG_CREDS;
+import static io.airbyte.integrations.source.bigquery.BigQuerySource.CONFIG_DATASET_ID;
+import static io.airbyte.integrations.source.bigquery.BigQuerySource.CONFIG_PROJECT_ID;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class BigQuerySourceEscapeColumnNameTest {
```

Comment:
As this test is copy of the `BigQuerySourceTest` test, please avoid copypasting. We can merge the or create an abstract class for such tests.